src/test/functional/udf.spec.ts

```diff
@@ -1,154 +1,167 @@
 import * as assert from "assert";
-import * as Stream from "stream";
-import {
-    AzureDocuments, Base, Constants, CosmosClient,
-    DocumentBase, HashPartitionResolver, Range,
-    RangePartitionResolver, Response, RetryOptions,
-} from "../../";
+import { CosmosClient } from "../../";
 import testConfig from "./../common/_testConfig";
 import { TestHelpers } from "./../common/TestHelpers";
 
-// Used for sproc
-declare var getContext: any;
-// declare var body: (input?: any) => void; // TODO: remove this if it's not necessary
-
-// TODO: should fix long lines
-// tslint:disable:max-line-length
-
-const host = testConfig.host;
+const endpoint = testConfig.host;
 const masterKey = testConfig.masterKey;
+const client = new CosmosClient({ endpoint, auth: { masterKey } });
+const dbId = "sample database";
+const containerId = "sample container";
 
-describe("NodeJS CRUD Tests", function () {
+describe("NodeJS CRUD Tests", function() {
     this.timeout(process.env.MOCHA_TIMEOUT || 10000);
-    // remove all databases from the endpoint before each test
-    beforeEach(async function () {
+
+    beforeEach(async function() {
         this.timeout(10000);
-        try {
-            await TestHelpers.removeAllDatabases(host, masterKey);
-        } catch (err) {
-            throw err;
-        }
+        // remove all databases from the endpoint before each test
+        await TestHelpers.removeAllDatabases(client);
```

Comment:
Per DM discussion, I think we can omit the try-catch blocks here and mocha will handle errors properly. I'll verify.